javascripts/discourse/initializers/init-force-home.gjs

```diff
@@ -0,0 +1,15 @@
+import { apiInitializer } from "discourse/lib/api";
+import { defaultHomepage } from "discourse/lib/utilities";
+
+export default apiInitializer("1.15.0", (api) => {
+  // anons can't do anything outside of home
+  // so force them back home
+  api.onPageChange(() => {
+    const router = api.container.lookup("service:router");
+    const currentUser = api.container.lookup("service:currentUser");
```

Comment:
service lookups use kebab-case (camelCase is a fallback, but for consistency and future compat it's better to stick to `-`)